### DISCUSSION TITLE
Welcome to bark.github.io Discussions!

### DISCUSSION THREAD

**Discussion by mariocraft987** [cytrinc/bark-coding#30] [Announcements]
## ![waving hand](https://u.cubeupload.com/fmji/1f44b.png) Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together ![flexed biceps](https://u.cubeupload.com/fmji/1f4aa.png).

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.![saluting face](https://u.cubeupload.com/fmji/1fae1.png)![saluting face](https://u.cubeupload.com/fmji/1fae1.png)

**Comment by raynepaws** [comment]
hooray! i was going to suggest you open discussions! :)

**Comment by mariocraft987** [comment]
nice
